### DISCUSSION TITLE
Lodestar Standup 2024-7 (February 20, 2024)

### DISCUSSION THREAD

**Discussion by nazarhussain** [ChainSafe/lodestar#6434] [Standups]
### Meeting Info

[Feb 20, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/feb-20-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
Led by @wemeetagain 
- Planning Items
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by nazarhussain** [comment]
We have two customer facing products `lodestar` and `prover`. We may introduce breaking changes in either e.g. changing some CLI flags. We need to discuss and decide the versioning scheme moving forward for both to highlight such scenarios.

**Reply by jeluard** [reply]
Relevant: https://github.com/ChainSafe/lodestar/pull/6428

**Reply by wemeetagain** [reply]
We discussed this in standup. Came to the following conclusion:
We should pull the prover package into its own repo so that we can version it independently of the rest of the lodestar libs.

**Comment by philknows** [comment]
Notes added here: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#february-21-2024-planning-and-standup-meeting

Closing in favour of #6467
